eclass/cmake-utils.eclass

```diff
@@ -441,6 +441,8 @@ cmake-utils_src_prepare() {
 	fi
 
 	popd > /dev/null || die
+
+	CMAKE_UTILS_SRC_PREPARE_HAS_RUN=1
```

Comment:
If you need packages to disable this, then give them an official explicit option for this purpose. Do not encourage developers to go playing around with internals. They do that enough without your help.